Automattic/vip-go-mu-plugins
# Updates convertor CLI tool to fetch DB name from DB

## Description
* the Previous implementation relies on having DB_NAME constant defined
which is not the case in prod deploy scenario

## Changelog Description

### CLI: utf8mb4 convertion

We updated the way the latin1 -> utf8mb4 conversion command retrieves the current DB name. To enable this command on more environments. 

## Checklist

Please make sure the items below have been covered before requesting a review:


## Steps to Test

1. Update the site 's meta db_charset=latin1 and db_collate=latin1_swedish_ci.
1. Regenerate wp-config.php
1. Run the command - `wp vip utf8mb4 convert --dry-run=true`
1. You should see: `Success: Set database to utf8mb4` as one of the last lines in the output.
1. Switch back charset so that your site is not broken :)
